core-java-modules/core-java-concurrency-basic-3/src/test/java/com/baeldung/concurrent/synchronizestatic/SychronizeStaticDataUnitTest.java

```diff
@@ -0,0 +1,96 @@
+package com.baeldung.concurrent.synchronizestatic;
+
+import org.junit.Test;
+
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
+
+/**
+ * The tests in this class show the output of creating multiple
+ * types of Employee classes in the <code>synchronizedstatic</code>
+ * package. When not synchronized the out will not be sequential;
+ * when it is synchronized the output will be in sequential.
+ */
+public class SychronizeStaticDataUnitTest
+{
+    private final Executor pool = Executors.newFixedThreadPool(4);
+
+    private final int numberToTest = 100;
+
+    @Test
+    public void whenNotSynchronized_thenDataOutOfOrder() {
+
+        System.out.println("No synchronization");
+
+        for(int i = 0; i < numberToTest; i++) {
+            int finalI = i;
+            pool.execute(() ->
+                         {
+                             new com.baeldung.concurrent.synchronizestatic.none.Employee(finalI, "John", "Smith");
+                         });
+        }
+    }
+
+    @Test
+    public void whenSynchronizedMethod_thenDataInOrder() {
+
+        System.out.println("Synchronization with synchronized method");
+
+        for(int i = 0; i < numberToTest; i++) {
+            int finalI = i;
+            pool.execute(() ->
+                         {
+                             new com.baeldung.concurrent.synchronizestatic.synchronizedmethod.Employee(finalI, "John", "Smith");
+                         });
```

Comment:
For all the blocks like this, we need to fix the curly brace placement and fix the indentation. It should only be four spaces over from the first character of the previous line.